praise-participants.md

```diff
@@ -0,0 +1,60 @@
+## Title
+Praise Participants
+
+## Patlet
+After an inner source contribution it's important to thank the contributor for their time and effort.
+This pattern gives guidance for a way to do so that not only effectively acknowledges the contribution but also endgenders further engagement from the contributor and others.
+
+## Problem
+How can we properly express our gratitude to a contributor for their inner source contribution to a project?
+It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
+A pattern in this area makes it easy to do and ensures that the message comes across clearly and sincerely.
+
+## Context
+* You are the trusted committer or maintainer on an inner source project.
+* You value the community of contributors and want to maintain and grow it.
+
+## Forces
+You are busy, which makes it easy to forget some of the soft touches like praise and thanks.
+You may not be someone that is comfortable in social situations or good with words.
+
+## Solutions
+It feels good to anyone to be recognized by others.
+In a professional setting, increased recognition is also an avenue to increased influence and growth.
+Any time someone gives to your inner source project, recognize them with a sincere "thank you".
```

Comment:
Instead of "sincere thank you" how about writing "qualified thank you"?

What I mean by qualified:

"I would like to thank you for writing up this pattern on praising participants, because it helped me internally make a case for the non-financial ways of motivating colleagues."

Explaining why the contributor gets a Thank You helps them understand what exactly they did that helped the project and determine what else to contribute next.

Reply:
https://youtu.be/h3MPewsk5PU?t=357 ... for where I'm coming from with "qualified thank you"

Reply:
`sincere and qualified "thank you"`